crates/hyperqueue/src/client/commands/submit/jobfile.rs

```diff
@@ -140,6 +141,7 @@ fn build_job_submit(jdef: JobDef, job_id: Option<JobId>) -> SubmitRequest {
             max_fails: jdef.max_fails,
         },
         submit_desc: JobSubmitDescription {
+            created_at: Utc::now(),
```

Comment:
This timestamp should be filled at the server, because clock on the server is the ground truth for event time.

So it should probably be a part of the `Job` in the server, not `JobSubmitDescription` (as something that is coming from the client).

Reply:
Hmm, good point. A bit annoying that this is *per submit*, so I'll need to store something like `SmallVec<[(Time, JobSubmitDescription); N]` on the server.

Reply:
Or we can have separate SmallVec with timestamps, but `SmallVec<[(Time, JobSubmitDescription); N]` is not nice but not that bad.

Reply:
Ok, change the storage to keep the submission date on the server.